<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>헤이딜러 로그인</string>
  </property>
  <widget class="QGroupBox" name="q_heydealer_login_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>300</width>
     <height>170</height>
    </rect>
   </property>
   <property name="title">
    <string>헤이딜러 로그인</string>
   </property>
   <widget class="QPushButton" name="q_heydealer_login_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>로그인</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>아이디</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="q_heydealer_id">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="q_headless_check">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Headless</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="q_heydealer_pw">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>비밀번호</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="q_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>헤이딜러에 로그인해주세요</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>q_heydealer_id</tabstop>
  <tabstop>q_heydealer_pw</tabstop>
  <tabstop>q_headless_check</tabstop>
  <tabstop>q_heydealer_login_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
